uniffi_bindgen/src/interface/mod.rs

```diff
@@ -754,7 +754,15 @@ impl ComponentInterface {
                     meta.index,
                 );
             }
-            cbi.methods.push(meta.into());
+            let defn: Method = meta.into();
+            for arg in defn.iter_types() {
+                self.types.add_known_type(arg);
+            }
+            if let Some(ty) = &defn.return_type {
```

Comment:
thanks - but `iter_types()` also gives the return type, so this block should be unnecessary

Reply:
How convenient! Adjusted, thanks